recipes/rgi/meta.yaml

```diff
@@ -61,6 +61,7 @@ requirements:
     - bamtools >=2.5.1
     - bedtools >=2.28.0
     - bwa >=0.7.17
+    - openssl ==1.0
```

Comment:
Please use `openssl 1.0.*` instead. If you see the log info, you will see that openssl 1.1.1d is installed due to the incorrect syntax.

Reply:
That's odd, I wonder what the difference between my command line is from the CI